vspace/src/test/java/edu/asu/diging/vspace/core/services/impl/ContentBlockManagerTest.java

```diff
@@ -84,6 +88,15 @@ public void test_deleteTextBlockById_forNonExistentId() throws BlockDoesNotExist
         Mockito.doThrow(EmptyResultDataAccessException.class).when(textBlockRepo).deleteById(textBlockId);
         managerToTest.deleteTextBlockById(textBlockId, "slideId_1");
     }
+    
+    @Test(expected = BlockDoesNotExistException.class)
+    public void test_deleteSpaceBlockById_forNonExistentId() throws BlockDoesNotExistException {
+        String spaceBlockId = "notARealId";
+        Optional<ContentBlock> contentBlockOptional = Optional.of(contentBlock);
+        when(contentBlockRepository.findById("notARealId")).thenReturn(contentBlockOptional);
+        Mockito.doThrow(EmptyResultDataAccessException.class).when(spaceBlockRepo).deleteById(spaceBlockId);
+        managerToTest.deleteSpaceBlockById(spaceBlockId,"slideId_1");
+    }
```

Comment:
what about adding/editing/deleting space blocks?

Reply:
?

Reply:
what about editing space blocks?

Reply:
where?

Reply:
<img width="1047" alt="Screenshot 2024-05-08 at 15 33 27" src="https://github.com/diging/virtual-spaces-2.0/assets/89062459/30931a86-ccc1-45db-a500-d51e92a1037d">

Reply:
I got so confused since the comment was so old 😅